testcases/crypto/aes_func.c

```diff
@@ -1561,6 +1606,14 @@ CK_RV do_WrapUnwrapRSA(struct generated_test_suite_info * tsuite)
         goto testcase_cleanup;
     }
 
+    if (!wrap_supported(slot_id, tsuite->mech)) {
+        testsuite_skip(3, "Slot %u doesn't support %s (0x%x)",
```

Comment:
Maybe make the text a bit more precise:
`"Slot %u doesn't support wrapping with %s (0x%x)"`
(it may support the mechanisms as such, but just not wrapping with it).